File: backend/src/main/java/com/app/mevocab/internal/quiz/dto/request/QuizSaveRequest.java
package com.app.mevocab.internal.quiz.dto.request;

import lombok.*;
import lombok.experimental.FieldDefaults;

@Builder
@Data
@AllArgsConstructor
@NoArgsConstructor
@FieldDefaults(level = AccessLevel.PRIVATE)
public class QuizSaveRequest {
    String quizID;
    String topic;
    String title;
}

File: frontend/lib/data/models/dto/Request/QuizSaveRequest.dart
class QuizSaveRequest {
  final String quizID;
  final String topic;
  final String title;
  final int totalTime;

  QuizSaveRequest({
    required this.quizID,
    required this.topic,
    required this.title,
    required this.totalTime,
  });

  Map<String, dynamic> toJson() {
    return {
      'quizID': quizID,
      'topic': topic,
      'title': title,
      'totalTime': totalTime,
    };
  }
}
